bigdata_service/src/main.py

```
@@ -5,10 +5,15 @@
 from aiokafka import AIOKafkaProducer
 from fastapi import FastAPI
 from fastapi.responses import ORJSONResponse
-from src.api import router as api_router
+from fastapi_pagination import add_pagination
+from motor.motor_asyncio import AsyncIOMotorClient
+
+from src.api import setup_routers
 from src.core.config import settings
 from src.core.logger import LOGGING, setup_root_logger
 from src.middleware.request_log import RequestLogMiddleware
+from src.core.logger import LOGGING
```

Ruff

Redefinition of unused `LOGGING` from line 13

Remove definition: `LOGGING`

(F811)